src/nvim/terminal.c

```diff
@@ -612,6 +612,11 @@ void terminal_get_line_attributes(Terminal *term, win_T *wp, int linenr,
   }
 }
 
+Buffer terminal_get_buffer(const Terminal *term)
```

Comment:
`terminal_get_buf`

We usually don't use "get" in C function names, it's implied, but there are several cases in this PR. `terminal_buf` would be the usual convention.